floss/floss.py

```diff
@@ -74,6 +74,7 @@ def stack_result(section: List[bytes]) -> Optional[ResultSection]:
     """ Generates a ResultSection from floss stacked strings output section """
     result = ResultSection('FLARE FLOSS Sacked Strings', body_format=BODY_FORMAT.MEMORY_DUMP,
                            heuristic=Heuristic(3))
+    assert result.heuristic
```

Comment:
won't this raise an AssertionError which isn't caught anywhere? https://www.w3schools.com/python/ref_keyword_assert.asp

Reply:
The assertion is being used to help the type checker. The result always has a heuristic because it was constructed with one (line 75) but the type checker can't figure it out. If any assertion fails the exception shouldn't be caught because it means something that the code assumed isn't true and so the code is definitely broken.